nuxt.config.ts

```diff
@@ -22,13 +22,13 @@ export default defineNuxtConfig({
   css: ['vuetify/styles'],
 
   modules: [
-    '@pinia/nuxt',
-    'nuxt-vitest',
     '@nuxtjs/i18n',
+    '@pinia/nuxt',
     'vuetify-nuxt-module',
+    'nuxt-vitest',
   ],
   i18n: {
-    lazy: true,
+    lazy: false,
```

Comment:
Hvorfor ikke lazy? Var det har bugen var?

Reply:
Yes, it's not working. Will look a little bit more at it.